modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -678,10 +678,10 @@ protected QueryFilter createFieldValueQueryFilter(
 
 		BooleanQuery booleanQuery = new BooleanQueryImpl();
 
-		Boolean localizableField = true;
+		Boolean localizableField = false;
 
 		if (ddmStructure.hasFieldByFieldReference(fieldReference)) {
-			localizableField = Boolean.valueOf(
+			localizableField = GetterUtil.getBoolean(
 				ddmStructure.getFieldPropertyByFieldReference(
 					fieldReference, "localizable"));
 
```

Comment:
Hi @davidtellolr,
Renato is on vacation so I'll be reviewing this PR.

Is it possible to order this alphabetically? That is, assign the value of `ddmStructureFieldValue` first?